app/MMB/src/components/exploreCity/ExploreButton.js

```diff
@@ -0,0 +1,43 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, TextButton } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+type Props = {|
+  +cityName: string,
+|};
+
+const onPress = () => {
+  console.warn('TODO');
+};
+
+export default function ExploreButton(props: Props) {
+  return (
+    <View style={styles.row}>
+      <View>
+        <TextButton
+          title={
+            <Translation
+              id="mmb.main_menu.explore_city.card_content.explore"
+              values={{ city: props.cityName }}
+            />
+          }
+          onPress={onPress}
+        />
+      </View>
+      <View style={styles.item} />
```

Comment:
What is this doing?

Reply:
Since our buttons by default takes the full width, this view takes up the width, so that the button does not.

Reply:
That feels a bit hacky and it is not entirely clear in the code itself. Could you please add a comment to explain this?